radix-engine-tests/tests/package.rs

```diff
@@ -643,6 +635,860 @@ fn publishing_of_package_with_a_lookalike_character_fails() {
     test_publishing_of_packages_with_invalid_names("depοsit")
 }
 
+#[test]
+fn test_error_path_when_package_definition_has_too_many_fields() {
+    // Arrange
+    let mut test_runner = TestRunnerBuilder::new().build();
+    let (code, mut definition) = PackageLoader::get("address");
+
+    definition.blueprints.values_mut().for_each(|bp_def| {
+        if let Some(field) = bp_def.schema.state.fields.first() {
+            bp_def.schema.state.fields = (0..300).map(|_| field.clone()).collect::<Vec<_>>();
```

Comment:
Maybe it would be worth to add a new constant in validation of the schema substates count in `package.rs` (now there is value 0xff), and then use that constant + 1 here.

Reply:
Good point! Will do that.